xbmc/platform/linux/SystemdUtils.cpp

```diff
@@ -0,0 +1,102 @@
+/*
+ *      Copyright (C) 2005-2013 Team XBMC
+ *      http://kodi.tv
+ *
+ *  This Program is free software; you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation; either version 2, or (at your option)
+ *  any later version.
+ *
+ *  This Program is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ *  GNU General Public License for more details.
+ *
+ *  You should have received a copy of the GNU General Public License
+ *  along with XBMC; see the file COPYING.  If not, write to
+ *  the Free Software Foundation, 675 Mass Ave, Cambridge, MA 02139, USA.
+ *  http://www.gnu.org/copyleft/gpl.html
+ *
+ */
+
+#include "SystemdUtils.h"
+
+#include "DBusUtil.h"
+#include "utils/log.h"
+
+// systemd DBus interface specification:
+// https://www.freedesktop.org/wiki/Software/systemd/dbus
+
+#define SYSTEMD_DEST  "org.freedesktop.systemd1"
+#define SYSTEMD_PATH  "/org/freedesktop/systemd1"
+#define SYSTEMD_IFACE "org.freedesktop.systemd1.Manager"
```

Comment:
Prefer `const std::string` over `#define` whenever possible

And yeah, I know it's c&p from the other code, just saying :-)